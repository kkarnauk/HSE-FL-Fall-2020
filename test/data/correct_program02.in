f f f :- (f, g, f); fff, (dd, f); f.
