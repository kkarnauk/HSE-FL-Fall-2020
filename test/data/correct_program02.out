|relation
|	|head
|	|	|atom
|	|	|	|identifier = 'f'
|	|	|	|identifier = 'f'
|	|	|	|identifier = 'f'
|	|body
|	|	|disjunction
|	|	|	|conjunction
|	|	|	|	|identifier = 'f'
|	|	|	|	|conjunction
|	|	|	|	|	|identifier = 'g'
|	|	|	|	|	|identifier = 'f'
|	|	|	|disjunction
|	|	|	|	|conjunction
|	|	|	|	|	|identifier = 'fff'
|	|	|	|	|	|conjunction
|	|	|	|	|	|	|identifier = 'dd'
|	|	|	|	|	|	|identifier = 'f'
|	|	|	|	|identifier = 'f'